### PR TITLE
Gdwr/nix

### PR DESCRIPTION
**Pull request by GDWR**
Add some utils for dev on nix

### CODE REVIEW

**Review by fisher60 — APPROVED** [review]
Polluting all of our repos 😭

File: flake.nix [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
+                packages = with pkgs; [
+                    nodejs_20
+                    nodePackages.yarn
+                    nodePackages.node-gyp
+                    vips
+                    python3
+                    poetry
+                    prisma-engines
+                    openssl
+		    pyright
```
[the remark this answers is not in the document]
Comment (GDWR) [reply flake.nix]: image

**Review by MrHemlock — APPROVED** [review]
Uses nix, but we're okay with that.  Approved

**Review by fisher60 — APPROVED** [review]
I have many regrets

**Pull request merged**